Sources/Sentry/SentryProfiler.mm

```diff
@@ -484,6 +484,7 @@ + (void)useFramesTracker:(SentryFramesTracker *)framesTracker
 + (nullable NSArray<SentrySample *> *)slicedSamples:(NSArray<SentrySample *> *)samples
                                         transaction:(SentryTransaction *)transaction
 {
+    [[SentryViewHierarchy alloc] init];
```

Comment:
h: I guess you forgot something here
```suggestion
```

Reply:
it's really annoying me that changes in a file under `Sources/` isn't triggering a new CI run 😡

Reply:
> it's really annoying me that changes in a file under Sources/ isn't triggering a new CI run 😡

@armcknight, the tests should run https://github.com/getsentry/sentry-cocoa/blob/230ba67c4f59d08a881b0fc0c791ed90b5de1419/.github/workflows/test.yml#L10

So does build https://github.com/getsentry/sentry-cocoa/blob/230ba67c4f59d08a881b0fc0c791ed90b5de1419/.github/workflows/build.yml#L10

Which parts of CI are not running? You can adjust this in the workflow files easily.

Reply:
aha: 
> If you use both the branches filter and the paths filter, the workflow will only run when both filters are satisfied

Reply:
So I just discovered that checks won't run if there is a merge conflict on the branch, that must've been what was going on: https://github.com/orgs/community/discussions/26304